.github/workflows/build_test_pr.yaml

```diff
@@ -22,6 +22,6 @@ jobs:
       env:
         PR: "${{ github.event.pull_request.number }}"
         SHA: "${{ github.event.pull_request.head.sha }}"
-        DOCKER_ACTOR: "${{ github.actor }}"
-        DOCKER_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        DOCKER_ACTOR: "jsfillman"
```

Comment:
So if I understand this right, all the Docker actions will be done as your username? And not a configured actor?

Reply:
Unfortunately, GitHub Container Registry can't "inherit" temporary auth tokens the way GitHub Package Registry can (yet).

This is a temporary solution until we can get a service account going to run CI independently of any specific user. Let's try to push for service account creation during Monday's standup, but it'll be a very easy find and replace to update this.